src/styles/_uswdsDependencies.scss

```diff
@@ -1,7 +1,10 @@
-@import 'node_modules/uswds/dist/scss/theme/uswds-theme-general';
-@import 'node_modules/uswds/dist/scss/theme/uswds-theme-typography';
-@import 'node_modules/uswds/dist/scss/theme/uswds-theme-spacing';
-@import 'node_modules/uswds/dist/scss/theme/uswds-theme-color';
-@import 'node_modules/uswds/dist/scss/theme/uswds-theme-utilities';
-@import './settings';
-@import 'node_modules/uswds/src/stylesheets/packages/_required.scss';
+// DEPRECATED AFTER USWDS 3.0 UPGRADE
+// #TODO REMOVE AFTER UPGRADE COMPLETE
+
+// @forward 'theme/styles';
+// @forward 'node_modules/uswds/dist/scss/theme/uswds-theme-typography';
+// @forward 'node_modules/uswds/dist/scss/theme/uswds-theme-spacing';
+// @forward 'node_modules/uswds/dist/scss/theme/uswds-theme-color';
+// @forward 'node_modules/uswds/dist/scss/theme/uswds-theme-utilities';
+// @forward './settings';
+// @forward 'node_modules/uswds/src/stylesheets/packages/_required.scss';
```

Comment:
Should we go ahead and remove this file?

Reply:
Yeah I think removing it makes sense